Source/WebInspectorUI/UserInterface/Views/RecordingActionTreeElement.js

```diff
@@ -55,9 +55,13 @@ WI.RecordingActionTreeElement = class RecordingActionTreeElement extends WI.Gene
             switch (swizzleType) {
             case WI.Recording.Swizzle.Number:
                 var constantNameForParameter = WI.RecordingAction.constantNameForParameter(recordingType, recordingAction.name, parameter, index, parameterCount);
+                var bitfieldNamesForParameter = WI.RecordingAction.bitfieldNamesForParameter(recordingType, recordingAction.name, parameter, index, parameterCount);
                 if (constantNameForParameter) {
                     parameterElement.classList.add("constant");
                     parameterElement.textContent = "context." + constantNameForParameter;
+                } else if (bitfieldNamesForParameter) {
+                    parameterElement.classList.add("constant");
+                    parameterElement.textContent = bitfieldNamesForParameter.map((p) => "context." + p).join("|");
```

Comment:
I think we should add some spaces `" | "` so that it appears a bit more readable

e.g. i personally think
```
context.COLOR_BUFFER_BIT | context.STENCIL_BUFFER_BIT
```
is easier to read than
```
context.COLOR_BUFFER_BIT|context.STENCIL_BUFFER_BIT
```